samples/subsys/video/capture_to_lvgl/src/main.c

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (c) 2024 Charles Dias <charlesdias.cd@outlook.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/device.h>
+#include <zephyr/drivers/display.h>
+#include <zephyr/drivers/video.h>
+#include <lvgl.h>
+
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(main);
+
+#define VIDEO_DEV_SW "VIDEO_SW_GENERATOR"
+
+void draw_label(void)
+{
+	lv_obj_t *label = lv_label_create(lv_scr_act());
+
+	lv_label_set_text(label, "LVGL on Zephyr!");
+	lv_obj_align(label, LV_ALIGN_CENTER, 0, 30);
+
+	lv_task_handler();
+}
+
+void draw_lvgl_logo(void)
+{
+	LV_IMG_DECLARE(img_lvgl_logo);
+	lv_obj_t *img1 = lv_img_create(lv_scr_act());
+
+	lv_img_set_src(img1, &img_lvgl_logo);
+	lv_obj_align(img1, LV_ALIGN_CENTER, 0, -10);
+
+	lv_task_handler();
+}
+
+void display_splash_screen(void)
+{
+	draw_label();
+	draw_lvgl_logo();
+
+	k_sleep(K_MSEC(5000));
+	lv_obj_clean(lv_scr_act());
+}
+
+int main(void)
+{
+	struct video_buffer *buffers[2], *vbuf;
+	const uint16_t WIDTH_VIDEO = CONFIG_VIDEO_WIDTH;
+	const uint16_t HEIGHT_VIDEO = CONFIG_VIDEO_HEIGHT;
```

Comment:
Is this idiomatic of how to do things in LVGL?
Would it work to use `CONFIG_VIDEO_WIDTH` everywhere instead of `VIDEO_WDITH`?

Reply:
You are totally right! Neither `WIDTH_VIDEO` nor `HEIGHT_VIDEO` is necessary. I'll update the sample.

Thanks, @josuah!